setup_env.sh

```diff
@@ -1,7 +1,33 @@
 #!/bin/bash                                                                                                         
 
-cp -R pathcheck-mobile-resources/environment/ ./
+DOWNLOAD_PATH=$1
```

Comment:
Is download path the best name here? Would authority_name be better?

Reply:
hm, maybe `ABBREVIATED_AUTHORITY`? It's used in the interaction between the GAEN server and the app (the GAEN server prefixes the url path of the key archives with this value), but I agree, since it's tied to the name of the HA, it would be clearer if that were reflected in the naming